cluster/common.sh

```diff
@@ -386,703 +306,6 @@ function find-release-tars() {
   fi
 }
 
-# Discover the git version of the current build package
-#
-# Assumed vars:
-#   KUBE_ROOT
-# Vars set:
-#   KUBE_GIT_VERSION
-function find-release-version() {
-  KUBE_GIT_VERSION=""
-  if [[ -f "${KUBE_ROOT}/version" ]]; then
-    KUBE_GIT_VERSION="$(cat ${KUBE_ROOT}/version)"
-  fi
-  if [[ -f "${KUBE_ROOT}/_output/release-stage/full/kubernetes/version" ]]; then
-    KUBE_GIT_VERSION="$(cat ${KUBE_ROOT}/_output/release-stage/full/kubernetes/version)"
-  fi
-
-  if [[ -z "${KUBE_GIT_VERSION}" ]]; then
-    echo "!!! Cannot find release version"
-    exit 1
-  fi
-}
-
-# Quote something appropriate for a yaml string.
-#
-# TODO(zmerlynn): Note that this function doesn't so much "quote" as
-# "strip out quotes", and we really should be using a YAML library for
-# this, but PyYAML isn't shipped by default, and *rant rant rant ... SIGH*
-function yaml-quote {
-  echo "'$(echo "${@:-}" | sed -e "s/'/''/g")'"
-}
-
-# Builds the RUNTIME_CONFIG var from other feature enable options (such as
-# features in alpha)
-function build-runtime-config() {
```

Comment:
This function was a no-op so I just deleted it as well.

Reply:
sounds good